docs/examples/generic_synchronization.rst

```diff
@@ -0,0 +1,25 @@
+Generic Synchronization Example
+===============================
+
+This example demonstrates how to synchronize two :any:`nixnet.session.FrameInStreamSession`
+session types on different interfaces.
+
+These principles can be extended to any session type and quantity.
+Sessions are created, configured, and started for each interface sequentially.
+
+The interfaces that will listen for a clock should be created and configured first.
+Interface properties need only be set once per interface. The connect_terminals function
+acts like an interface property and needs to be set only once per interface. Listening
+sessions can then be started with the "Session Only" scope. The last session to be started
+on a particul interface can be started with the normal scope which will cause it to start
```

Comment:
particular*

Might be nice to actually link to docstrings for things like "connect_terminals", "SESSION_ONLY", and any other relevant properties.

If possible, it's a good idea to get input from techcomms on docstrings.